frontend/src/routes/+page.svelte

```diff
@@ -1,17 +1,64 @@
 <script>
-
+import { onMount } from 'svelte';
     const storage_url = import.meta.env.VITE_STORAGE_URL
     /** @type {import('./$types').PageData} */
     export let data;
 
+    onMount(() => {
+        const carousel = document.querySelector('.carousel');
+        const carouselItems = carousel.querySelectorAll('.carousel-item');
+
+        let isDragging = false;
+        let startPosX = 0;
+        let scrollLeft = 0;
+
+        carousel.addEventListener('mousedown', (e) => {
+        isDragging = true;
+        startPosX = e.clientX - carousel.offsetLeft;
+        scrollLeft = carousel.scrollLeft;
+        });
+
+        carousel.addEventListener('mousemove', (e) => {
+        if (!isDragging) return;
+        const mouseX = e.clientX - carousel.offsetLeft;
+        const deltaX = mouseX - startPosX;
+        carousel.scrollLeft = scrollLeft - deltaX;
+        });
+
+        carousel.addEventListener('mouseup', () => {
+        isDragging = false;
+        });
+
+        carousel.addEventListener('mouseleave', () => {
+        isDragging = false;
+        });
+
+        const carouselItemWidth = carouselItems[0].getBoundingClientRect().width;
+        const scrollLeftButton = document.querySelector('#carousel-btn-left');
+        const scrollRightButton = document.querySelector('#carousel-btn-right');
+
+        let scrollItemNumber = 1
+        if (window.innerWidth > 768) scrollItemNumber = 3
+
+        scrollLeftButton.addEventListener('click', () => {
+        carousel.scrollLeft -= carouselItemWidth * scrollItemNumber; // Ajusta la cantidad de desplazamiento según sea necesario
```

Comment:
Do you feel that this comment is really useful info? If you do, then translate it to English.
I'd remove it, otherwise, or put it above the `scrollItemNumber` variable to avoid the duplication (if I understood it correctly, that's the value you'd edit).